mk/update-travis-yml.py

```diff
@@ -111,6 +115,12 @@ def format_entries():
             %(sources)s"""
 
 def format_entry(os, target, compiler, rust, mode, features):
+    if target == "x86_64-fortanix-unknown-sgx" and rust != "nightly":
```

Comment:
When will this target be in stable Rust?

Reply:
It will land in beta if https://github.com/rust-lang/rust/pull/57659 is accepted for backporting, which will put it in stable at 1.33.0. However, that version won't include the fixes for LTO and building on Ubuntu 14.04.